imports/client/components/PuzzleListPage.tsx

```diff
@@ -346,6 +347,52 @@ interface PuzzleListPageTracker {
   hunt: HuntType;
 }
 
+const StyledPuzzleListLinkList = styled.ul`
+  list-style: none;
+  display: flex;
+  align-items: stretch;
+  flex-wrap: wrap;
+  width: 100%;
+  margin: 0 0 8px 0;
+  padding: 0;
+  border-color: #cfcfcf;
+  border-style: solid;
+  border-width: 1px 0;
+`;
+
+const StyledPuzzleListLink = styled.li`
+  display: flex;
+  align-items: stretch;
+  flex: 1 1 0;
+`;
+
+const StyledPuzzleListLinkAnchor = styled(Link)`
+  flex: 1 1 0;
+  display: flex;
+  height: 38px;
+  align-items: center;
+  align-content: center;
+  justify-content: center;
+  text-align: center;
+  padding: 8px 0;
+  font-size: 14px;
+  font-weight: bold;
+  &:hover {
+    background-color: #f8f8f8;
+  }
+`;
+
+const StyledPuzzleListExternalLink = styled(StyledPuzzleListLink)`
+  flex: 0 0 40px;
+`;
+
+const StyledPuzzleListLinkLabel = styled.span`
+  margin-left: 4px;
+  @media (max-width: ${() => window.getComputedStyle(document.body).getPropertyValue('--breakpoint-sm')}) {
```

Comment:
Just out of curiosity: how often does this get computed?  Seems like the sort of thing that could wind up causing JS to block on DOM operations and lead to jank if it's every render or something.

Reply:
As far as I can tell, it's only evaluated on load. Right now there are 3 calls to `mediaBreakpointDown` so it gets called 3 times. We could theoretically build out some caching here, but I worry that we'd have to think about, like, load order and stuff like that and would mildly prefer to just not, but I can look into it if you think it's important.